s3/src/main/scala/akka/stream/alpakka/s3/scaladsl/S3Client.scala

```diff
@@ -132,7 +139,7 @@ final class ObjectMetadata private (
    * @see ObjectMetadata#setContentType(String)
    */
   lazy val contentType: Option[String] = metadata.collectFirst {
-    case h if h.lowercaseName() == "content-type" => h.value
+    case ct: `Content-Type` => ct.value
```

Comment:
Following the boy scout rule, I improved this.

There are several Content-Type classes: `akka.http.scaladsl.model.ContentType`, `akka.http.scaladsl.model.headers.Content-Length` and before #858 we were using the wrong one...